.symfony.bundle.yaml

```diff
@@ -8,5 +8,5 @@ maintained_branches:
     - "3.5.x"
     - "3.6.x"
     - "3.7.x"
-doc_dir: "docs/"
+doc_dir: {"3.5.x": "Resources/doc/", "3.6.x": "docs/", "3.7.x": "docs/"}
```

Comment:
```suggestion
doc_dir: "docs/"
```

Reply:
OK, I added another commit to remove `3.5.x` from maintained versions. Thanks.

Reply:
Thank you!